media/plg_media-action_smartcrop/js/smartcrop.js

```diff
@@ -79,11 +84,36 @@ Joomla.MediaManager.Edit = Joomla.MediaManager.Edit || {};
 		return basePath;
 	}
 
+	function addCustomWidths() {
+		var widths = Joomla.getOptions('js-smartcrop-widths');
+		var widthDropDown = document.getElementById("jform_requestedWidth");
+		var defaultWidthLength = widthDropDown.length;
+		var customWidthLength  = widths.length;
+		if (customWidthLength > 0) {
```

Comment:
Always keep a blank line before control structures (a blank line is a line that contains an opening brace { or nothing).